backend/app/tests/utils/utils.py

```
@@ -93,7 +95,42 @@
+            raise ValueError(
+                f"Missing the following PostgreSQL credentials in {env_test_path}: {', '.join(missing_vars)}"
+            )
+
+    else:
+        raise FileNotFoundError(
+            f"Warning: {env_test_path} not found. No environment variables loaded."
+        )
+
+    return settings
+
+  
```

Ruff (0.12.2)

Blank line contains whitespace

(W293)